R/standalone-stringr.R

```diff
@@ -40,5 +44,45 @@ str_detect <- function(string, pattern, fixed = FALSE) {
   grepl(pattern = pattern, x = string, fixed = fixed)
 }
 
+str_replace <- function(string, pattern, replacement){
+  sub(x = string, pattern = pattern, replacement = replacement)
+}
+
+str_replace_all <- function (string, pattern, replacement){
```

Comment:
Can we also add a `str_replace_all(fixed)` argument that we pass to `gsub(fixed)`?